hw/ip/tlul/data/tlul.prj.hjson

```diff
@@ -14,14 +14,14 @@
         design_stage:       "D3",
         verification_stage: "V3",
         commit_id:          "0078a3228c0d311cd40996a730ed8453640a6944",
-        notes:              "the target for this entry is the autogenerated xbar_main within top_earlgrey."
+        notes:              "The target for this entry is the autogenerated xbar_main within top_earlgrey."
       }
       {
         version:            "1.0",
         life_stage:         "L1",
         design_stage:       "D1",
         verification_stage: "V1",
-        notes:              ""
+        notes:              "Use both FPV and DV to perform block level verification."
```

Comment:
I would rephrase this to: "Use FPV for the TL building blocks and DV for the generated XBARs."

Reply:
Thanks that sounds good, i will update the notes!